bangazonreports/urls.py

```diff
@@ -1,6 +1,7 @@
 from django.urls import path
-from .views import expensive_list
+from .views import expensive_list, Cheap_list, completed_order_list
 
 urlpatterns = [
     path('reports/expensive', expensive_list),
-]
+    path('reports/cheap', Cheap_list),
+    path('reports/orders/completed', completed_order_list)
```

Comment:
Reseeding fails because of this line. `SyntaxError: unexpected EOF while parsing`.
Be sure to close the `]` and retest this this code.